app/views/hq/orphans/_form.html.erb

```diff
@@ -116,7 +116,7 @@
       </div>
       <div class="form-group">
         <%= f.label_for_field :guardian_id_num %>
-        <%= f.text_field :guardian_id_num, class: "form-control" %>
+        <%= f.number_field :guardian_id_num, class: "form-control" %>
```

Comment:
Yes, I agree. But I got an error related to this field in the form view validation. It complains that the view doesn't have the field: https://github.com/AgileVentures/osra/blob/develop/spec/views/hq/orphans/_form_spec.rb#L47